content/guides/nextjs/adding-tina/creating-forms.md

```diff
@@ -63,7 +62,7 @@ First, we'll need to import `useForm` and `usePlugin` from the `tinacms` package
 import { useForm, usePlugin } from 'tinacms'
 ```
 
-Now, just add the form to the `Post` component with the configuration we laid out previously:
+Then, just add the form to the `Post` component with the configuration we laid out previously:
```

Comment:
```suggestion
Then, add the form to the `Post` component with the configuration we laid out previously:
```

Reply:
The whole reason for this change was to get rid of that word and then I didn't